package.json

```
@@ -125,6 +130,12 @@
                             "description": "Any additional args you want to use for that sigmac"
                         }
                     }
+                },
+                "sigma.sigconverterBackend": {
+                    "type": "string",
+                    "default": "splunk",
+                    "enum": ["splunk", "loki","microsoft365defender", "opensearch","qradar","sentinelone","sentinelone_pq"],
```

As these are related to the pysigma version installed. It would not be ideal to hardcode them. Its fine for now but just FYI

I'll make sure to add an API endpoint to list backends soon.
```suggestion
                    "enum": ["splunk", "loki","microsoft365defender", "opensearch","qradar","sentinelone","sentinelone_pq"],
```